packages/httph/httph.0.1/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+synopsis: "Minimal OCaml to the httpserver.h http server toolkit"
+description: """
+A very simple OCaml http toolkit intended to be a very straightforward set of bindings to the underlying
+httpserver.h C library.
+"""
+maintainer: "Travis Brady <travis.brady@gmail.com>"
+authors: "Travis Brady <travis.brady@gmail.com>"
+license: "MIT"
+homepage: "https://github.com/travisbrady/ocaml-httph"
+bug-reports: "https://github.com/travisbrady/ocaml-httph/issues"
+dev-repo: "git://git@github.com:travisbrady/ocaml-httph.git"
+depends: [ "ocaml" "ocamlfind" ]
+build: [
+  ["./configure" "--prefix=%{prefix}%"]
+  [make]
+]
+install: [make "install"]
```

Comment:
Your project seems to be using dune but dune is not defined as a dependency nor is it used to build the library, and thus fails at the first step.

Reply:
Thanks, @kit-ty-kate. I believe I've got this fixed in https://github.com/travisbrady/ocaml-httph/commit/bd53775811a24b2038bcf0e82632af42da0ba71d but am not sure how to test locally. Is there an opam command that I can use to verify correctness before proceeding? thank you

Reply:
`opam install .` inside the repository should do it

Reply:
I've also commented on the commit you linked above

Reply:
Ok thank you. I rm'd that install file and `opam install .` looks good.
Should I re-initiate this somehow with `opam publish` again? Sorry not sure of the workflow for fixing a broken PR

Reply:
Calling `opam publish` again works.